crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -1457,6 +1469,17 @@ pub mod tests {
         }
     }
 
+    fn cluster_configuration(nodes: u8) -> ClusterConfiguration {
```

Comment:
`nodes` is a bit misleading since it seems to be the replication factor.

Reply:
True. Will fix